### ISSUE TITLE
Color output

### ISSUE THREAD

**Issue by nizartaha** [ibois-epfl/script-sync#45]
**Describe the bug**
the component outputs Color from a list of dictionaries as `{'color': <System.Drawing.Color object at 0x0000027557785800>}`

my input is colour CMYK component from grasshopper which is added to a list of dictionaries `list = [{'color': input_color}]`

**Expected output**
`{'color': (0,0,0)} `or `{'color': [A=255, R=0, G=0, B=0]} `

**OS**
 - OS: Windows
 - Rhino 8

**Additional bug**
bug which is indirectly related
script-sync fails if input is a list with more than 2 items

**Comment by 9and3** [comment]
Hello @nizartaha thanks for reporting this!

To be able to help you could you please provide:
- the exact Rhino version (go to "help >> about Rhinoceros.." and see the small number on the right of the poped up window
- could you add a `.ghx` and/or the code you are trying to runnin?

Thanks!

**Comment by nizartaha** [comment]
Hello @9and3 

Here you go. Thank you for the plugin

- Version 8 SR12 (8.12.24282.7001, 2024-10-08)
- github doesnt support .ghx so here is the script for a color_bug.py file and here a screenshot to be able to reproduce from grasshopper.
- please note that i included another issue with the file related to a list of lists output.

![image](https://github.com/user-attachments/assets/7d9821b9-81cf-4340-b143-d636f98a8621)

```
# 1 _ bug 1 color
data = [
    {
        "type": "list of dictionaries",
        "color": [color_CMYK]
    }
]

# 2 _ bug 2
x = pt.X
y = pt.Y
z = pt.Z
s = ["Hello World"]
lxyz = [x, y, z]

out = [s]
out2 = [lxyz]
# please comment out the following line
# out3 = [x, y, z, s]
```

**Comment by 9and3** [comment]
Hello @nizartaha ,

So I finally got some time to have a look at this issue:

* 1) Concerning the color output of a dictionnary. This is a normal behavior that you see the `System.Drawing.Color` object since this is how Grasshopper internalize color objects. If you want to visualize it in the pannel as you described you can use a magic method `__str__()`, like this:
![image](https://github.com/user-attachments/assets/2a13d4a1-b006-4668-a970-193d2b6b2c6a)

```py
data = [
    {
        "type": "list of dictionaries",
        "color": [color_CMYK.__str__()]
    }
]
```

* 2) About the lists it is indeed a bug, it is solved in #46 and a new script-sync release is already out. If you close and reopen your Rhino should be fine.

I am closing this one, and thanks again for the bug report and using script-sync!
